### PR TITLE
TOC Restructuring project - 6.6 content for review

### PR DESCRIPTION
**Pull request by ghost**
This PR contains content from 6.6 that needs to be included in 7.0.

### CODE REVIEW

File: modules/fts/6_6_content/fts-supported-queries-compound-query.adoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[#compound-queries]
```
Comment (RayOffiah) [inline modules/fts/6_6_content/fts-supported-queries-compound-query.adoc]: There's no real need for anchors at the top of the page.

File: modules/fts/6_6_content/fts-supported-queries-geo-bounded-polygon.adoc [hunk 2]
```diff
@@ -0,0 +1,68 @@
+[#creating_geospatial_rest_query_polygon_based]
```
Comment (RayOffiah) [inline modules/fts/6_6_content/fts-supported-queries-geo-bounded-polygon.adoc]: Not usually required

File: modules/fts/6_6_content/fts-supported-queries-geo-bounded-polygon.adoc [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+= Creating a Query: Polygon-Based
+
+The following query-body uses an array, each of whose elements is a string, containing two floating-point numbers; to specify the latitude and longitude of each of the corners of a polygon &#8212; known as _polygon points_.
+In each string, the `lat` floating-point value precedes the `lon.`
+
+Here, the last-specified string in the array is identical to the initial string, thus explicitly closing the box.
+However, specifying an explicit closure in this way is optional: closure will be inferred by Couchbase Server if not explicitly specified.
```
Comment (RayOffiah) [inline modules/fts/6_6_content/fts-supported-queries-geo-bounded-polygon.adoc]:
```suggestion
However, specifying an explicit closure in this way is optional: the closure will be inferred by Couchbase Server if not explicitly specified.
```

File: modules/fts/6_6_content/fts-supported-queries-geo-bounded-polygon.adoc [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+If a target data-location falls within the box, its document is returned.
+The results are specified to be sorted on `name` alone.
+
+[source,javascript]
```
Comment (RayOffiah) [inline modules/fts/6_6_content/fts-supported-queries-geo-bounded-polygon.adoc]:
```suggestion
[source, json]
```

File: modules/fts/6_6_content/fts-supported-queries-geo-bounded-polygon.adoc [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+  },
+  "sort": [
+    "name"
+  ]
+}
+----
+
+A subset of formatted output might appear as follows:
+
+[source,javascript]
```
Comment (RayOffiah) [inline modules/fts/6_6_content/fts-supported-queries-geo-bounded-polygon.adoc]:
```suggestion
[source, json]
```

File: modules/fts/6_6_content/fts-supported-queries-geo-bounded-rectangle.adoc [hunk 6]
```diff
@@ -0,0 +1,70 @@
+[#creating_geospatial_rest_query_bounding_box_based]
```
Comment (RayOffiah) [inline modules/fts/6_6_content/fts-supported-queries-geo-bounded-rectangle.adoc]: Doubt this is required

File: modules/fts/6_6_content/fts-supported-queries-geo-bounded-rectangle.adoc [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+= Creating a Query: Rectangle-Based
+
+In the following query-body, the `top_left` of a rectangle is expressed by means of an array of two floating-point numbers, specifying a longitude of `-2.235143` and a latitude of `53.482358`.
+The `bottom_right` is expressed by means of key-value pairs, specifying a longitude of `28.955043` and a latitude of `40.991862`.
+The results are specified to be sorted on `name` alone.
+
+[source,javascript]
```
Comment (RayOffiah) [inline modules/fts/6_6_content/fts-supported-queries-geo-bounded-rectangle.adoc]:
```suggestion
[source, json]
```

File: modules/fts/6_6_content/fts-supported-queries-geo-bounded-rectangle.adoc [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+  },
+  "sort": [
+    "name"
+  ]
+}
+----
+
+A subset of formatted output might appear as follows:
+
+[source,javascript]
```
Comment (RayOffiah) [inline modules/fts/6_6_content/fts-supported-queries-geo-bounded-rectangle.adoc]:
```suggestion
[source, json]
```

File: modules/fts/6_6_content/fts-supported-queries-geo-point-distance.adoc [hunk 9]
```diff
@@ -0,0 +1,82 @@
+[#creating_geospatial_rest_query_radius_based]
```
Comment (RayOffiah) [inline modules/fts/6_6_content/fts-supported-queries-geo-point-distance.adoc]: Suspect this is not needed

File: modules/fts/6_6_content/fts-supported-queries-geo-point-distance.adoc [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+= Creating a Query: Radius-Based
+
+This section and those following, provide examples of the query-bodies required to make geospatial queries with the Couchbase REST API.
+Note that more detailed information on performing queries with the Couchbase REST API can be found in xref:fts-searching-with-the-rest-api.adoc[Searching with the REST API]; which shows how to use the full `curl` command and how to incorporate query-bodies into it.
+
+The following query-body specifies a longitude of `-2.235143` and a latitude of `53.482358`.
+The target-field `geo` is specified, as is a `distance` of `100` miles: this is the radius within which target-locations must reside for their documents to be returned.
+
+[source,javascript]
```
Comment (RayOffiah) [inline modules/fts/6_6_content/fts-supported-queries-geo-point-distance.adoc]:
```suggestion
[source, json]
```

File: modules/fts/6_6_content/fts-supported-queries-geo-point-distance.adoc [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+    }
+  ]
+}
+----
+
+The query contains a `sort` object, which specifies that the returned documents should be ordered in terms of their _geo_distance_ from specified `lon` and `lat` coordinates: these values need not be identical to those specified in the `query` object.
+
+A subset of formatted console output might appear as follows:
+
+[source,javascript]
```
Comment (RayOffiah) [inline modules/fts/6_6_content/fts-supported-queries-geo-point-distance.adoc]:
```suggestion
[source, json]
```

File: modules/fts/6_6_content/fts-supported-queries-non-analytic-query.adoc [hunk 12]
```diff
@@ -0,0 +1,18 @@
+[#non-analytic-queries]
```
Comment (RayOffiah) [inline modules/fts/6_6_content/fts-supported-queries-non-analytic-query.adoc]: Probably not required

File: modules/fts/6_6_content/fts-supported-queries-range-query.adoc [hunk 13]
```diff
@@ -0,0 +1,10 @@
+[#range-queries]
```
Comment (RayOffiah) [inline modules/fts/6_6_content/fts-supported-queries-range-query.adoc]: I don't think you need this.

File: modules/fts/6_6_content/fts-supported-queries-term-range.adoc [hunk 14]
```diff
@@ -0,0 +1,18 @@
+[#term-range]
```
Comment (RayOffiah) [inline modules/fts/6_6_content/fts-supported-queries-term-range.adoc]: Heading anchors are rarely needed.

File: modules/fts/6_6_content/fts-supported-queries-special-query.adoc [hunk 15]
```diff
@@ -0,0 +1,9 @@
+[#special-queries]
```
Comment (RayOffiah) [inline modules/fts/6_6_content/fts-supported-queries-special-query.adoc]: No need for heading anchor.

**Review by RayOffiah — CHANGES_REQUESTED** [review]
Only use anchors if you're going to link to this heading from the current page or another one, otherwise it's just extra work you don't need to do.

File: modules/fts/6_6_content/fts-supported-queries-compound-query.adoc (hunk 1 above)
Comment (ghost) [reply modules/fts/6_6_content/fts-supported-queries-compound-query.adoc]: removed anchor

File: modules/fts/6_6_content/fts-supported-queries-geo-bounded-polygon.adoc (hunk 2 above)
Comment (ghost) [reply modules/fts/6_6_content/fts-supported-queries-geo-bounded-polygon.adoc]: removed anchor

File: modules/fts/6_6_content/fts-supported-queries-geo-bounded-polygon.adoc (hunk 3 above)
Comment (ghost) [reply modules/fts/6_6_content/fts-supported-queries-geo-bounded-polygon.adoc]: updated

File: modules/fts/6_6_content/fts-supported-queries-geo-bounded-polygon.adoc (hunk 4 above)
Comment (ghost) [reply modules/fts/6_6_content/fts-supported-queries-geo-bounded-polygon.adoc]: updated

File: modules/fts/6_6_content/fts-supported-queries-geo-bounded-polygon.adoc (hunk 5 above)
Comment (ghost) [reply modules/fts/6_6_content/fts-supported-queries-geo-bounded-polygon.adoc]: updated

File: modules/fts/6_6_content/fts-supported-queries-geo-bounded-rectangle.adoc (hunk 6 above)
Comment (ghost) [reply modules/fts/6_6_content/fts-supported-queries-geo-bounded-rectangle.adoc]: removed anchor

File: modules/fts/6_6_content/fts-supported-queries-geo-bounded-rectangle.adoc (hunk 7 above)
Comment (ghost) [reply modules/fts/6_6_content/fts-supported-queries-geo-bounded-rectangle.adoc]: updated

File: modules/fts/6_6_content/fts-supported-queries-geo-bounded-rectangle.adoc (hunk 8 above)
Comment (ghost) [reply modules/fts/6_6_content/fts-supported-queries-geo-bounded-rectangle.adoc]: updated

File: modules/fts/6_6_content/fts-supported-queries-geo-point-distance.adoc (hunk 9 above)
Comment (ghost) [reply modules/fts/6_6_content/fts-supported-queries-geo-point-distance.adoc]: removed anchor

File: modules/fts/6_6_content/fts-supported-queries-geo-point-distance.adoc (hunk 10 above)
Comment (ghost) [reply modules/fts/6_6_content/fts-supported-queries-geo-point-distance.adoc]: updated

File: modules/fts/6_6_content/fts-supported-queries-geo-point-distance.adoc (hunk 11 above)
Comment (ghost) [reply modules/fts/6_6_content/fts-supported-queries-geo-point-distance.adoc]: updated

File: modules/fts/6_6_content/fts-supported-queries-non-analytic-query.adoc (hunk 12 above)
Comment (ghost) [reply modules/fts/6_6_content/fts-supported-queries-non-analytic-query.adoc]: removed anchor

File: modules/fts/6_6_content/fts-supported-queries-range-query.adoc (hunk 13 above)
Comment (ghost) [reply modules/fts/6_6_content/fts-supported-queries-range-query.adoc]: removed anchor

File: modules/fts/6_6_content/fts-supported-queries-term-range.adoc (hunk 14 above)
Comment (ghost) [reply modules/fts/6_6_content/fts-supported-queries-term-range.adoc]: removed anchor

File: modules/fts/6_6_content/fts-supported-queries-special-query.adoc (hunk 15 above)
Comment (ghost) [reply modules/fts/6_6_content/fts-supported-queries-special-query.adoc]: removed anchor

**Comment by ghost** [comment]
Closing this PR as its updates are merged into PR 2402.

**Pull request closed without merging**